src/deploy/RPM_build/noobaa.spec

```diff
@@ -26,14 +26,18 @@ URL:  https://www.noobaa.io/
 Source0:  %{noobaatar}
 
 BuildRequires:  systemd
-BuildRequires:  python3.9
 BuildRequires:  make
 BuildRequires:  gcc-c++
 BuildRequires:  boost-devel
 BuildRequires:  libcap-devel
 %if 0%{?rhel} == 8
 BuildRequires:  gcc-toolset-11
 %endif
+%if 0%{?rhel} == 10
+BuildRequires:  python3.12
+%else
+BuildRequires:  python3.9
```

Comment:
I am not sure why it was `python3.9` instead of just `python3` but I think it was probably because one of the built-in module had different behaviour in older python version than in 3.9 🤔 .

Reply:
It might not even get built with python version older than 3.9 (a lot of projects already dropped support for older versions). The default version in rhel 8 is older than 3.9 though so that is why we probably needed this.

Reply:
Yeah @b-ranto, I think that was the reason we fixed it earlier.